Automattic/wp-calypso
# Implemented the new mailbox action menu

This feature will enable rendering the mailbox actions as an ellipsis action menu.

Based on the type of Mailbox and available actions, either Gmail's or Titan's action menu is being shown to the user.

#### Changes proposed in this Pull Request

* Added a Mailbox action menu consisting of email-related actions for Titan and Gmail.

#### Testing instructions



1. Open the branch on your local server and go to `Upgrades` -> `Domains.`
2. Click any email account to open its Domain view
3. Click on "Manage your email accounts."
4. Click on the `EllipsisMenu` on the right of your mailbox to see.

**Before:**
![image](https://user-images.githubusercontent.com/1598017/118120747-fdc43180-b40d-11eb-98cd-4c933710c618.png)

**After (Gmail Action Menu)**
![image](https://user-images.githubusercontent.com/1598017/118120814-17657900-b40e-11eb-8261-bc875c60c618.png)

**After (Titan Action Menu)**
![image](https://user-images.githubusercontent.com/1598017/118120879-35cb7480-b40e-11eb-9e84-ca85eda0a3f0.png)

cc: @daledupreez
